src/NuGet/Microsoft.CodeAnalysis.Workspaces.Common.nuspec

```diff
@@ -29,10 +29,10 @@ Supported Platforms:
   <files>
     <file src="Microsoft.CodeAnalysis.Workspaces.dll" target="lib\portable-net45+win8" />
     <file src="Microsoft.CodeAnalysis.Workspaces.xml" target="lib\portable-net45+win8" />
```

Comment:
That is the correct NuGet behavior, at least in terms it's doing what we're asking it to do. portable-net45+win8 doesn't mean "install me only into portable libraries" it means "I'm a portable library that can be installed into platforms like net45 (or higher) or win8 (or higher.)"

I'm not sure portable-net46+win8 exists, at least in reasonable sense of it existing. I do agree this behavior is pretty bad though. I think we need to consider breaking this into two separate packages, since this is just too subtle what's going on. Thankfully we can use a 2.0 jump per semver for that.